config/main.py

```diff
@@ -570,6 +570,82 @@ def is_ipaddress(val):
         return False
     return True
 
+#
+# Check if an interface_name is in a vlan
+#
+def  interface_is_in_vlan(vlan_member_table, interface_name):
+
+    for k,v in vlan_member_table:
```

Comment:
No. here we are checking the key. by default it checks the keys, but I modified as vlan_member_table.keys() to make it explicit.

Reply:
Ah, I think I was confused because of the `k, v` variable names. Could you make the variable name more descriptive? I didn't realize the keys themselves were split into two parts. :)